### ISSUE TITLE
Technical Information Wanted, Boeing 707-320C and 707-420

### ISSUE THREAD

**Issue by Megaf** [Megaf/707#30]
Hi all,

We are looking for maintenance manuals or official specs for the 707, either from Boeing or from Airline companies.
Please submit here PDFs, images, charts, schematics and so on.
Below is a list of what we need ASAP.

- [ ] 707 Fuel System Diagrams and Schematics
- [ ] 707 Correct start procedure
- [ ] Hydraulic diagrams
- [ ] Pneumatic diagrams

I already have some stuff at https://github.com/Megaf/707/issues/18

Thanks.

**Comment by FirstOfficerDelta** [comment]
Did you try this: https://pdfcoffee.com/boeing-707-300-operations-manual-pdf-free.html?

**Comment by Megaf** [comment]
Yep, we are actually using that manual. It is basically the only freely available manual on the internet.
